web/src/ts/components/forms/LoginForm.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FC, useState } from 'react'
+
+import { useActions } from '../../hooks/useActions'
```

Comment:
It is not cool that you should return func `useAction`
which has func inside `const { loginUser } = useActions()`